scripts/addAuctionTimestamp.sql

```diff
@@ -0,0 +1 @@
+UPDATE parcels SET auction_timestamp = 1517356800000 where auction_owner IS NOT NULL AND auction_price IS NOT NULL;
```

Comment:
Should we add this query to the migration up? how can we run it?

Reply:
you're right, historically we've been running `.sql` files directly on the server, but this should be done each time the migration runs, I'll change it 👍